package.json

```diff
@@ -9,11 +9,17 @@
   "devDependencies": {
     "jest": "^23.4.1",
     "prettier": "^1.13.7",
-    "prettier-check": "^2.0.0"
+    "prettier-check": "^2.0.0",
+    "watchify": "^3.11.0"
   },
   "scripts": {
     "test": "jest",
     "lint": "prettier-check src/*.js",
-    "format": "prettier --write src/*.js"
+    "format": "prettier --write src/*.js",
+    "watch": "watchify src/main.js --verbose --standalone envSpec -o dist/envspec.js",
+    "start": "browser-sync start --server demo --server dist --watch --files 'dist/*' 'demo/*'  --port $PORT"
+  },
+  "dependencies": {
+    "browser-sync": "^2.24.6"
```

Comment:
We need `browser-sync` to be in the `devDependencies`, since it is a tool used by the developers of `env-spec-js` (us) and is not necessary for the users of `env-spec-js` itself.

You can fix this by running the following commands:

```
yarn remove browser-sync
yarn add --dev browser-sync
```